src/components/Datepicker/Month.tsx

```diff
@@ -61,7 +62,7 @@ const Month: FC<MonthProps> = ({ year, month, firstDayOfWeek, locale }: MonthPro
                         return <Day date={day.date} key={day.date.toString()} dayLabel={day.dayLabel} />;
                     }
 
-                    return <div key={day} />;
+                    return <div key={generateId()} />;
```

Comment:
This fixes the warning but doesn't solve the issue 100% from the React perspective, I guess it's not crazy computation, but it will rerender all the cells every time because is a new id every time.

What about using indexes? is going to receive the same index for the first empty cells, until it gets the date for the actual first day of the month. It's a bit ugly I guess, but would work in this case I think.

Maybe I'm being too picky and it wouldn't add any value, what do you think?

Reply:
I'm not sure the key would affect the amount of rerenders since there are no changes to the order of the items in the list, the only moment where it will have to rerender is when the parent rerenders (when the Datepicker is opened or the user changes the month). 
But I do think as well that for this case indexes would work, since the order does not change the indexes can always uniquely identify the items. I used a generated id because this is the foolproof way of setting a key to a list item when the data does not already have an id, but if everyone agrees I can change it to use the indexes.